Formula/nest.rb

```diff
@@ -0,0 +1,152 @@
+class Nest < Formula
+  desc "The Neural Simulation Tool (NEST) with Python2 bindings (PyNEST)"
+  homepage "http://www.nest-simulator.org/"
+  url "https://github.com/nest/nest-simulator/archive/v2.14.0.tar.gz"
+  sha256 "afaf7d53c2d5305fac1257759cc0ea6d62c3cebf7d5cc4a07d4739af4dbb9caf"
+  head "https://github.com/nest/nest-simulator.git"
+
+  bottle do
+    sha256 "4f7d765cd75ae3272ce15bb293c20a6cdfc9ccb551f0a07d00016c75ae0ef267" => :high_sierra
+    sha256 "60e4b0595c5d11003ffe1eca3fdca3c197b8711336ead21eba67620db96454c0" => :sierra
+    sha256 "8a4c8e6d0d2e4ce93d76af73197c693f8c303095800e82b7266b922848f1a39b" => :el_capitan
+    sha256 "6207dea13f90b17a95313b964323f7971cd5e2507a38b0ffd5d64b2f346e7851" => :x86_64_linux
+  end
+
+  option "with-python3", "Build Python3 bindings (PyNEST) instead of Python2 bindings."
+  option "without-openmp", "Build without OpenMP support."
+  needs :openmp if build.with? "openmp"
```

Comment:
don't need this option

Reply:
I do not understand, why we don't need this option? nest can be build with or without OpenMP. Are you suggesting, we build with OpenMP by default?

Reply:
I'm saying we should have no openmp option.